tensorflow/lite/micro/examples/network_tester/network_model.h

```diff
@@ -16,6 +16,119 @@ limitations under the License.
 #ifndef TENSORFLOW_LITE_MICRO_EXAMPLES_NETWORK_TESTER_NETWORK_MODEL_H_
 #define TENSORFLOW_LITE_MICRO_EXAMPLES_NETWORK_TESTER_NETWORK_MODEL_H_
 
+#ifdef ETHOS_U
+alignas(16) const unsigned char network_model[] = {
```

Comment:
Is it because bazel can not run downloaded models? Anyhow it seems like bazel is not running network tester so that should be ok.

Could we not download a model and convert it without overwriting the original model? I think that is what we would have to do for the benchmarks that is using the downloaded person detect model. Or do you mean download, transform and not keeping the original model so network tester will only run for Ethos-U with the default model?

Assuming we could download a model, convert it without overwriting the original, could we not use the person detect int8 model that is already downloaded?
It will take less than 2 seconds more to run network tester on bluepill with person detect instead of the current default model. Or should/could we download some smaller model (keyword model used in benchmarks?)?

Reply:
I was thinking that we remove the existing model in network_tester completely (to avoid any confusion).

What you are saying makes sense (w.r.t benchmarks). For this PR, let's use the person detection model, converting it and using from network tester for Ethos-U without changing the behavior (or model) of network_tester for any other target.

And let's open a ticket to figure out how we want to do this for benchmarks and we can plan to remove network_tester as well once we get benchmarks going with converted models.

Reply:
Ok

Reply:
BTW we have this ticket: https://github.com/tensorflow/tensorflow/issues/47071